docs/getting-started-guides/docker.md

```diff
@@ -55,35 +55,7 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
-2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
I'm fine with moving this to ```Troubleshooting``` section. I just don't want to discourage people by seeing and reading a long and complicated list of kernel options.